package.json

```diff
@@ -66,6 +66,7 @@
     "@vitejs/plugin-react": "^4.3.4",
     "ajv": "^8.17.1",
     "babel-plugin-formatjs": "^10.5.34",
+    "dompurify": "^3.2.4",
```

Comment:
I think we need to list this one in the `dependencies` rather than `peerDependencies` since it's a hard requirement.